### PR TITLE
Refs #17798 - Update wording on CV change on single/bulk content-host

### PR DESCRIPTION
**Pull request by chris1984**
Bulk action change:

https://nimbusweb.me/s/share/4456621/z6mt7177qger59o1955w

Single host change:

https://nimbusweb.me/s/share/4456539/8n42nya0kdnz17bc0xf2

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html [hunk 1]
```diff
@@ -3,11 +3,6 @@ <h4 data-block="modal-header" translate>Content Host Bulk Content</h4>
 
   <div data-block="modal-body">
     <h4 translate>Assign Lifecycle Environment and Content View</h4>
-    <p>Please run the following commands on the client, so the new Content View can take effect.</p>
-    <pre>
-    # yum clean all
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:8]: I'm not sure if `yum clean all` is actually required. Keep in mind that not all content hosts are using yum.

cc @sthirugn

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html [hunk 2]
```diff
@@ -63,6 +58,10 @@ <h4 translate>Assign Lifecycle Environment and Content View</h4>
   </div>
 
   <div data-block="modal-footer">
+    <p style="text-align:left;font-size:12px;"><b>Note:</b> Updating the content view here will not automatically update the client. </p>
```
Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:61]: I don't think we need to word 'Note' - it's already in a tooltip :)

Comment (jturel) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:61]: I think the message is a little bit unclear. It almost makes it sound as if this is required, when it's really only required if you want it to take effect immediately.

What about "Changing the Content View will not affect the host until its next checkin. To update the host immediately run the following commands"


cc @sthirugn

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html (hunk 1 above)
Comment (sthirugn) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:8]: @jturel you are right, `yum clean all` is not really required. I also just tested it to make sure.
```
[root@client01 ~]# cat /etc/yum.repos.d/redhat.repo | grep baseurl
baseurl = https://sat-6-7-qa-rhel7.apex.example.com/pulp/repos/Default_Organization/QE/cv1/custom/zooproduct/zoorepo
```
<at this point, I updated the content view from cv1 to cv2 in Satellite UI for this client>

```
[root@client01 ~]# subscription-manager refresh
2 local certificates have been deleted.
All local data refreshed
[root@client01 ~]# cat /etc/yum.repos.d/redhat.repo | grep baseurl
baseurl = https://sat-6-7-qa-rhel7.apex.example.com/pulp/repos/Default_Organization/PROD/cv2/custom/zooproduct/zoorepo
```

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html (hunk 2 above)
Comment (sthirugn) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:61]: @jturel 
- I think you may be right, but I don't have time to validate this.  Are you sure about this or do we need to validate this?
- Assuming the above is validated, I like your messaging, maybe just capitalize `Host`

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html (hunk 1 above)
Comment (chris1984) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:8]: Removed `yum clean all`

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html (hunk 2 above)
Comment (chris1984) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:61]: True, removed :)

Comment (chris1984) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/views/content-hosts-bulk-environment-modal.html:61]: Fixed and updated the wording.

**Comment by chris1984** [comment]
@jturel updated, thanks for the review.

@sthirugn thanks for looking at it 👍

**Comment by jturel** [comment]
@chris1984 I was experimenting with the styling a bit - so I just opened up a PR against your branch: https://github.com/chris1984/katello/pull/1

**Comment by sthirugn** [comment]
thank you @chris1984 and @jturel

**Pull request merged**